datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/config/AppConfigResolver.java

```diff
@@ -188,12 +193,24 @@ public CompletableFuture<AppConfig> get(final DataFetchingEnvironment environmen
       }
       visualConfig.setTheme(themeConfig);
     }
-    if (_visualConfiguration != null && _visualConfiguration.getApplication() != null) {
+    if (_settingsService != null) {
```

Comment:
this used to be powered by the env variable - I'm changing this here to be powered by the switch instead.